### DISCUSSION TITLE
Importing of Nessus results

### DISCUSSION THREAD

**Discussion by spectreltd** [NUWCDIVNPT/stig-manager#346] [Wishlist]
Similar to Vulnerator. Parsing of nessus files.

**Comment by csmig** [comment]
Nessus vulnerability scans are very distinct from STIG assessments. They are out of scope for this STIG management tool. Thanks for the request, perhaps you are proposing a different tool? I believe Vulnerator is a POA&M management capability, which certainly could be implemented as a separate REST API by interested parties!
